File: Haskell/built-in/built_in_function.hs
reverseNum :: Int -> Int
reverseNum num = read(reverse(show(num)))::Int

reverseNumX :: String -> String
reverseNumX num = reverse(num)

main = do
	print(reverseNum 1234567890987654321)
	print(reverseNumX "123456789098765432123456789")

File: Java/built-in/built_in_function.java
import java.util.Scanner;

public class Reverse
{
	public String reverseNum(String num)
	{
		StringBuilder str = new StringBuilder(num);
		str.reverse();
		return str.toString();
	}
}

class ReverseNumber
{
	public static void main(String[] args)
	{
		Scanner in = new Scanner(System.in);
		Reverse rev = new Reverse();
		String num = in.next();
		String res = rev.reverseNum(num);
		System.out.println(res);
	}
}